README.md

```diff
@@ -7,66 +7,47 @@
-
-Semgrep releases [new versions very frequently](https://github.com/returntocorp/semgrep/releases) and the rules in this repo are always updated to take advantage of new features or fix breaking changes on new releases. For that reason, we suggest that you avoid stale copies of registry rules.
+Run existing and custom Semgrep rules locally with the Semrgep command line interface (Semgrep CLI) or continuously with Semgrep in CI while using Semgrep App. To start using Semgrep rules, see [Semgrep tutorial](https://semgrep.dev/learn), [Getting started with Semgrep CLI](https://semgrep.dev/docs/getting-started/), and [Getting started with Semgrep App](https://semgrep.dev/docs/semgrep-app/getting-started-with-semgrep-app/).
 
 ## Contributing
 
-We welcome Semgrep rule contributions directly to this repository! Since this repo is maintained by r2c, there are some extra benefits-for example, if there are bug reports for your rule, we’ll take responsibility to help fix it. If you are submitting to the semgrep-rules repo (rather than your own, separate repository as mentioned above) we’ll ask you to make r2c a joint owner of your contributions. While you still own copyright rights to your rule, joint ownership allows r2c to license these contributions to other [Semgrep Registry](https://semgrep.dev/r) users pursuant to the LGPL 2.1 under the [Commons Clause](https://commonsclause.com/). Check out the [Contributing Guidelines](/CONTRIBUTING.md) to get started.
-
-You can also contact us at support@r2c.dev to make Semgrep rule contributions. We will import your rules for everyone to use!
-
-### Rulesets
-
-Rulesets -- combined sets of rules from the Semgrep registry -- are organized in a private repository. If you want to modify existing sets or create your own, please contact us at support@r2c.dev.
-
-If you have more questions, please see the [FAQ section in the Semgrep docs](https://semgrep.dev/docs/faq).
-
-## Help
+We welcome Semgrep rule contributions directly to this repository! If you are submitting to the `semgrep-rules` repository we’ll ask you to make r2c a joint owner of your contributions. While you still own copyright rights to your rule, joint ownership allows r2c to license these contributions to other [Semgrep Registry](https://semgrep.dev/r) users pursuant to the LGPL 2.1 under the [Commons Clause](https://commonsclause.com/). Full [license details](https://github.com/returntocorp/semgrep-rules/blob/develop/LICENSE) here.
 
-Join [Slack](https://r2c.dev/slack) for the fastest answers to your questions! Or contact the team at [support@r2c.dev](mailto:support@r2c.dev).
+:::info
+To contribute, find details about contributing in the **[Contributing to Semgrep rules](https://semgrep.dev/docs/contributing/contributing-to-semgrep-rules-repository/)** documentation.
+:::
 
-### Rule Namespacing
-
-The namespacing format for contributing rules is `<language>.<framework>.<category>.$MORE`. If a `framework` isn't applicable, use `lang` instead.
```

Comment:
Shouldn't we leave this part in?

Reply:
Also, the part about `lang` in the namespace means that we do use languages to categorize rulesets (see last line below).

Reply:
We do have this info in the docs, so I'm open to removing it here? I think it's better to keep these important details all in the docs, so there's one source of truth for users, so it's harder for things to be out of date or in multiple places. What do you think?

Reply:
Ohh so that's why we're linking to contributing-to-semgrep-rules. Got it! Yes this works for me. I will resolve it. Thank you @milanwilliams !